launch.rb

```diff
@@ -42,7 +42,7 @@ def self.require_fastlane_ci
 
     def self.verify_app_built
       if ENV["WEB_APP"]
-        app_exists = File.file?(File.join('public', '.dist', 'index.html'))
+        app_exists = File.file?(File.join("public", ".dist", "index.html"))
```

Comment:
@nakhbari before submitting, do a quick `bundle exec rubocop -a` 😄

Reply:
Shoot sorry about that. You're supposed to use single tick in JS/TS got used to it